src/components/js/EcApplication.js

```diff
@@ -194,5 +264,21 @@ export default {
       },
       immediate: true
     }
+  },
+
+  created () {
+    for (var i = 0; i < this.tabListNoTitle.length; i++) {
+      switch (this.tabListNoTitle[i].key) {
+        case 'description':
+          this.tabListNoTitle[0].tab = this.i19description
+          break;
+        case 'settings':
+          this.tabListNoTitle[1].tab = this.i19configuration
+          break;
+        case 'related':
+          this.tabListNoTitle[2].tab = this.i19relatedApps
+          break;
+      }
+    }
```

Comment:
Continua repetitivo. Acho que poderia ser só um foreach setando a tab com o i19 tipo assim;

``` javascript
created () {
    this.tabListNoTitle.forEach((tab, index) => {
      this.tabListNoTitle[index].tab = this[`i19${tab.key}`]
    })
  }
```

Mas se for utilizar esse foreach é necessario alterar o nome da key das tabs pra `description`, `configuration` e `relatedApps`. E também alterar no EcApplication.html o if que verifica qual é a `activeTabKey`.

Reply:
Nesse caso, não daria pois eu não quero fixar um termo x, eu quero que de acordo com o lang seja x ou y. Se eu fixar lá no key, ficará sempre em ingles, mas estou alterando aqui para ficar melhor de outra forma

Reply:
Não, ele não vai ficar sempre ingles. Ele sempre vai buscar o termo conforme a _lang() setado no config. De qualquer forma as coisas precisam de nomes, o que eu quis dizer com a sugestão é que deixando o nome das key parecido com o nome que voce definir pros termos computados te da a possibilidade de deixar o código mais dinâmico, como no exemplo.